### PR TITLE
Update references to New Relic's language-specific exporters for OpenTelemetry

### PR DESCRIPTION
**Pull request by jgals**
See DOC-6641 for details.

This PR updates references to New Relic's language-specific exporters for OpenTelemetry in favor of the collector and OTLP.

### CODE REVIEW

**Comment by urbiz-nr** [comment]
Thanks for the updates, @jgals - could you sign the CLA so I can approve the changes?

**Comment by jgals** [comment]
Thank you, @urbiz-nr! I have signed the CLA and will wait on a content review from you and @a-feld at this point.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes.mdx [hunk 1]
```diff
@@ -48,7 +48,7 @@ If you set up the collector as a sidecar, OpenTelemetry calls this an agent:
 
 ## Tail-based sampling [#tail-based]
 
-By default, OpenTelemetry uses head-based sampling, which means traces are randomly sampled up front before details about the completed traces are known. As an alternative, you can configure OpenTelemetry to use our tail-based sampling, which analyzes all your traces after they're completed. If you want to use tail-based sampling, set up the collector to send every span to a trace observer in New Relic Edge where they're analyzed.
+By default, OpenTelemetry uses head-based sampling, which means traces are randomly sampled up front before details about the completed traces are known. As an alternative, you can configure OpenTelemetry to use our tail-based sampling, which analyzes all your traces after they're completed. If you want to use tail-based sampling, set up the collector to send every span to a trace observer in New Relic Edge where they're analyzed. For more information, see [Tail Sampling Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor).
```
Comment (a-feld) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes.mdx:51]: I think this needs to be a separate option. There are 2 options here getting shmoosed into 1 😄 

Options:
1. Use infinite tracing tail-based sampling
2. Use OpenTelemetry-Collector based tail-based sampling

Comment (jgals) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes.mdx]: Thanks for the feedback, @a-feld. I've separated these options for tail-based sampling.

**Comment by urbiz-nr** [comment]
Hi @jgals, @a-feld - is this ready for review? Thanks!

**Review by a-feld — APPROVED** [review]

**Comment by jgals** [comment]
Hi @urbiz-nr, yes, this is ready for review. Thank you!

**Review by urbiz-nr — APPROVED** [review]

**Review by urbiz-nr — APPROVED** [review]

**Review by urbiz-nr — APPROVED** [review]

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx [hunk 2]
```diff
@@ -159,19 +159,19 @@ APM agents offer detailed transaction trace visibility for individual services.
 
 ## How OpenTelemetry works with New Relic [#how-it-works]
 
-Here are two basic ways you could set up OpenTelemetry with New Relic: use the OpenTelemetry collector or use a New Relic exporter. Each approach assumes you set up instrumentation in your code, but they differ in how the data is gathered and sent to New Relic.
+Here are two basic ways you could set up OpenTelemetry with New Relic: use the OpenTelemetry collector (recommended), or use the native OTLP exporter (pre-release). Each approach assumes you set up instrumentation in your code, but they differ in how the data is gathered and sent to New Relic.
```
Comment (RichVanderwal) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:162]: The OpenTelemetry project's OTLP exporters for each language are in various states of development, and at least Java and Python's are GA. Perhaps you mean that it's our New Relic *OTLP endpoint* that's in pre-release?

File: src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx [hunk 3]
```diff
@@ -17,15 +17,7 @@ OpenTelemetry currently supports two telemetry types: distributed traces and met
...
-
-While it introduces more complexity, you can send your data through the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector). To do so, configure your OpenTelemetry SDK `service.name` as a resource and configure the SDK to send data to the collector. Then configure the collector with the [New Relic exporter for Go](https://github.com/newrelic/opentelemetry-exporter-go) with your Insert API key. Consult the [collector documentation](https://opentelemetry.io/docs/collector/about/) if you're having trouble sending data through the collector.
+To send your data through the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector), configure your OpenTelemetry SDK `service.name` as a resource and configure the SDK to send data to the collector. Then configure the collector with the [New Relic exporter for Go](https://github.com/newrelic/opentelemetry-exporter-go) with your Insert API key. Consult the [collector documentation](https://opentelemetry.io/docs/collector/about/) if you're having trouble sending data through the collector.
```
Comment (RichVanderwal) [inline src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx:20]: The link for the "New Relic exporter for Go" points to the Exporter that's being deprecated. Link to the [New Relic Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/newrelicexporter) in the collector instead -- these are two different pieces of software.

In fact, two paragraphs above, the phrase "If you're using a New Relic OpenTelemetry exporter" makes it sound like this is a troubleshooting guide for the exporters we're deprecating. Does it make sense to have this page at all?
Pasted_Image_4_22_21__8_49_AM

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx (hunk 2 above)
Comment (jgals) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:162]: Yes, good point @rvanderwal-newrelic, I'llfollow-up with a fix PR to replace _OTLP exporter_ with _OTLP endpoint_.

Comment (jgals) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:162]: https://github.com/newrelic/docs-website/pull/1975

File: src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx (hunk 3 above)
Comment (jgals) [reply src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx:20]: Thanks for the clarification, @rvanderwal-newrelic. I will update these links in a follow up PR.

Comment (jgals) [reply src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx:20]: #1975

**Pull request merged**